serial.md

```diff
@@ -103,8 +105,8 @@ specified by `maxBytesToRead` then the lesser number of bytes will be returned.
 ```
 0  START_SYSEX        (0xF0)
 1  SERIAL_DATA        (0x60)
-2  SERIAL_READ        (0x30) // OR with port (0x31 = SERIAL_READ | HW_SERIAL1)
-3  SERIAL_READ_MODE   (0x00) // 0x00 => read continuously, 0x01 => stop reading
+2  SERIAL_READ        (0x30) // OR with read mode: 0x30 => read continuously, 0x31 => stop reading
```

Comment:
There is an additional change to notice here. I've OR'd the read mode with byte 2. Another approach is to create a separate mode for each of the read modes (only 2 now but could be more in the future). In this case the modes would be:
```
SERIAL_READ_CONTINUOUSLY     (0x30)
SERIAL_READ_STOP             (0x32)
SERIAL_SOME_FUTURE_READ_MODE (0x32)
```
Reserving a range from 0x30 to 0x38 or 0x3F.

Another way to handle this would be set the read mode in byte 4, followed by `maxBytesToRead`, etc.